### PR TITLE
Static file globs

### PR DESCRIPTION
**Pull request by gauntface**
R: @jeffposnick @addyosmani 

First PR on moving over to sw-precache.

globPatterns => staticFileGlobs

Will add the CLI flag for this once sw-build has the rename in place - just for consistency.

### CODE REVIEW

**Comment by jeffposnick** [comment]
The code changes look fine. From a naming perspective, do you prefer `globPatterns` to `staticFileGlobs`? Because if so, instead of abandoning `globPatterns` everywhere, we could just accept `staticFileGlobs` as an alias to `globPatterns`, but encourage new developers to use `globPatterns`.

**Comment by addyosmani** [comment]
Imo, both `globPatterns` and `staticFileGlobs` are both equally understandable but open to aliasing depending on what Matt feels works best here.

File: packages/sw-cli/src/index.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ class SWCli {
         rootDirPath, fileExtentionsToCache);
       return swBuild.generateFileManifest({
         rootDirectory: rootDirPath,
-        globPatterns: [globPattern],
+        staticFileGlobs: [globPattern],
         globIgnores: [
           path.join(rootDirPath, fileManifestName),
         ],
```
Comment (addyosmani) [inline packages/sw-cli/src/index.js]: All 👍

**Comment by gauntface** [comment]
Naming I don't care about at this point to be honest.

**Comment by gauntface** [comment]
I'm going to fix the Windwos build as I have a hunch on where the issue but not 100%.

**Comment by jeffposnick** [comment]
👍

**Comment by addyosmani** [comment]
All 💚

**Pull request merged**